GPU/Vulkan/FramebufferVulkan.cpp

```diff
@@ -537,9 +626,7 @@ void FramebufferManagerVulkan::RebindFramebuffer() {
 }
 
 void FramebufferManagerVulkan::ResizeFramebufFBO(VirtualFramebuffer *vfb, u16 w, u16 h, bool force) {
-	return;
-
-	/*
+	// Copy the whole framebuffer object? Probably not a great idea.
```

Comment:
The reason for this, at least before, was to reuse `BlitFramebuffer`.  Needed to have the old vfb to copy from.

This is needed, for example, in Tales of Phantasia iirc to avoid graphical artifacts during skits.

-[Unknown]